config.yaml

```diff
@@ -9,4 +9,4 @@ options:
   service-account:
     type: string
     description: The service account to be used by driver to create executor pods.
-    default: 'kyuubi'
\ No newline at end of file
+    default: 'kyuubi-spark-engines'
```

Comment:
Adding to this, since the config option is not respected by the charm anymore (that's what I came to conclusion after looking at the code because we don't read it from config anywhere), can't we simply remove this altogether?

Reply:
I believe we have discussed what this config option is for, e.g. specifying the name for the service account to be requested to integration hub. 

Do we agree to keep it?

Reply:
Yes, let's keep it. The plural name for the default value seems a bit weird to my eyes, though.

Reply:
oh, yes. I can make it singular. I agree, it reads better